appendix-basic65.tex

```diff
@@ -5751,43 +5894,39 @@ \subsection{LINE INPUT\#}
 % ****
 % LIST
 % ****
-\label{basic65-list}
+\label{BASIC 65 Commands!LIST}
 \newpage
 \subsection{LIST}
 \index{BASIC 65 Commands!LIST}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$9B
 \item [Format:] {\bf LIST} [{\bf P}] [line range]
+\item [Usage:] Lists a range of lines from the BASIC program in memory.
 
-\item [Usage:] Used to list a range of lines from the BASIC program.
+               Given a single line number, {\bf LIST} lists that line.
 
-               {\bf line range} consists of the first and/or last
-               line to list, or a single line number.
-               If the first number is omitted, the
-               first BASIC line is assumed.
-               If the second number is omitted, the last BASIC line
-               is assumed.
+               Given a range of line numbers, {\bf LIST} lists all lines in
+               that range. A range can be two numbers separated by a hyphen
+               (\screentext{-}), a starting number and a hyphen, or a hyphen
+               and an ending number.
```

Comment:
I feel like we should clarify the impact of omitting the starting range or ending range numbers.

Reply:
Took a crack at it.